### ISSUE TITLE
Skills test

### ISSUE THREAD

**Issue by OleksiyRudenko** [py-gang/git-workshop#26]
The aim of this test is git flow skills test. That comprises:
- working in a feature branch
- updating the feature branch with changes from remote `dev`
- using rebase technique
- resolving a conflict
- merging changes via a Pull Request
- doing code review with requesting changes and approving

Do not use GitHub web UI for any changes, the only allowed operations in GitHub web UI are PR operations and code review operations.

Make sure completing every task using a dedicated feature branch.
Your changes merged into `dev` must form linear history.
Your commit messages must start with your GitHub username to track your authorship.

Should you require any help or support ask for some in the chat you use to communicate with your mentors and peers.

Complete the following:

1. Implement changes requested by a reviewer:
- add some changes to any file under your personal directory and commit
- open a PR
- ask someone to conduct a review and request changes
- implement changes and update your PR
- ask the reviewer to review, approve and merge into `dev` (using regular merge to preserve commits' hashes)
- report the task completed in the chat
- remove the feature branch locally and on remote once your job is approved by a mentor
Make sure you branch off of appropriate and up-to-date commit/branch in the repo
The reviewer is expected to suggest changes in your file.
The reviewer must be someone different from a peer you conducted a PR review for in her or his passing of this test.

2. Updating your feature branch with fresh changes from `dev` resolving a conflict
- add some changes to `./hello.world` and commit
- ask mentors to change `./hello.world`; wait for the requested change done
- update your feature branch with fresh changes from `dev`
- resolve a conflict: changes from `dev` must prevail but also add a comment denoting that there was a conflict (`Conflict resolved by YourGitHubUserName`) and commit
- open a PR
- ask someone to conduct a review, approve and merge into `dev` (using regular merge to preserve commits' hashes)
- report the task completed in the chat
- remove the feature branch locally and on remote once your job is approved by a mentor
Make sure you branch off of appropriate and up-to-date commit/branch in the repo
The reviewer must be someone different from a peer you conducted a PR review for in her or his passing of this test.

3. Get your GitHub sticker

**Q**. Why not reviewing PRs reciprocally, i.e. to each other doing the test in pairs?
**A**. To handle odd number of participants.
